src/components/resume-items/education-item.tsx

```diff
@@ -54,7 +78,7 @@ export function EducationItem() {
   return (
     <Dialog>
```

Comment:
We will need a place to display errors. How they do it here is 1 option [https://ui.shadcn.com/docs/components/form](url), or it might be easier to have 1 message location rather than a message location for every field